docs/src/extend/custom-rules.md

```diff
@@ -606,14 +606,19 @@ You can also access comments through many of `sourceCode`'s methods using the `i
 
 ### Options Schemas
 
-Rules may export a `schema` property, which is a [JSON Schema](https://json-schema.org/) format description of a rule's options which will be used by ESLint to validate configuration options and prevent invalid or unexpected inputs before they are passed to the rule in `context.options`.
+Rules may export a `schema` property, which is a [JSON Schema](https://json-schema.org/learn/) format description of a rule's options which will be used by ESLint to validate configuration options and prevent invalid or unexpected inputs before they are passed to the rule in `context.options`.
```

Comment:
I'd prefer that we keep the `https://json-schema.org/` homepage link here, as it describes what the JSON Schema is and gives an overview.